src/common/utils.js

```diff
@@ -107,12 +107,12 @@ export function formatNumberThousands(num) {
     return `${decimalStr}${period}${floatStr}`;
 }
 
-export const getProxyAccount = (api, sourceParaId, address, { addressType, multiLocationType } = { addressType: 'Substrate' }) => {
+export const getProxyAccount = (api, sourceParaId, address, { addressType, multiLocationType, network } = { addressType: 'Substrate' }) => {
     const locationType = multiLocationType || 'XcmV2MultiLocation';
-    const network = 'Any';
+    const networkType = network || 'Any';
```

Comment:
For consistancy, you should set the default value the same way as the addressType
`= { addressType: 'Substrate', multiLocationType: 'XcmV2MultiLocation', network: 'Any'}`

By the way, what is the reason to make the last three parameters an object, `{ addressType, multiLocationType, network }`? These are parameters necessary for this function. In my opinion we’d better flatten them out, for instance, `(api, sourceParaId, address, addressType = "Substrate", multiLocationType = "XcmV2MultiLocation" , network="Any" )`?